src/main/kotlin/xyz/atrius/waystones/commands/WarpstoneCommand.kt

```diff
@@ -0,0 +1,71 @@
+package xyz.atrius.waystones.commands
+
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import org.bukkit.Bukkit
+import xyz.atrius.waystones.utility.defaultWarpKey
+import xyz.atrius.waystones.utility.translateColors
+import xyz.atrius.waystones.utility.pluralS
+
+object WarpstoneCommand : CommandExecutor {
+    // CommandExecutor / Subcommand Manager
+    override fun onCommand(
+        sender: CommandSender,
+        cmd: Command,
+        lbl: String,
+        args: Array<String>
+    ): Boolean {
+        // Plugin Info
+        if (args.isEmpty())
+            return infoCmd(sender)
+
+        // Register Give WarpKey
+        val getkeyAliases = arrayOf("getkey", "gk", "warpkey", "wk", "key", "k")
+        if (args[0].toLowerCase() in getkeyAliases)
+            return getkeyCmd(sender, args.drop(1))
+
+        // Command Not Found
+        else sender.sendMessage("&d[Waystones]&r Unknown command, check &a/waystones&r".translateColors('&'))
+        return true
+    }
+
+    // Command - Plugin Info
+    private fun infoCmd (sender: CommandSender): Boolean {
+        val plugin = Bukkit.getPluginManager().getPlugin("Waystones")
+        sender.sendMessage("------------ &dWaystones&r ------------".translateColors('&'))
+        sender.sendMessage("Version: &a${plugin?.description?.version}".translateColors('&'))
+        sender.sendMessage("Author: &dAtriusX&r".translateColors('&'))
+        sender.sendMessage("&aCommands: &r".translateColors('&'))
+        sender.sendMessage("&a/waystones getkey &e[amount] [player]&r - Gives player set amount of keys".translateColors('&'))
```

Comment:
Would it be possible to convert this to a multi-line string and just translate/send once? This feels a bit needlessly repetitive. Example:
```kt
sender.sendMessage("""
	|------------ &dWaystones&r ------------
	|Version: &a${plugin?.description?.version}
	|Author: &dAtri&r
	|Commands: &r
	|&a/waystones getkey &e[amount] [player]&r - Gives player set amount of keys
""".trimMargin().translateColors('&'))